tests/components/modbus/conftest.py

```diff
@@ -79,7 +78,6 @@ async def run_base_read_test(
     now,
 ):
     """Run test for given config."""
-
```

Comment:
This has nothing to do with changing constants (which I see as a good idea), the empty line is design,  please remove this change.